.circleci/getChangelogs.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+REQUIREMENTS_FILE="codacy/requirements.yaml"
```

Comment:
Following your suggestion @paulopontesm , helm dep up does not propagate the git info that exists on requirements.yaml on to requirements.lock. 
We must use the git info stored in the requirements.yaml after all.